llvm/lib/Target/RISCV/RISCVFeatures.td

```diff
@@ -716,6 +716,13 @@ def FeatureStdExtSmaia
                        "AIA specifies for a hart, over all privilege levels.)",
                        []>;
 
+def FeatureStdExtSmepmp
```

Comment:
Can we keep smaia and ssaia together since they come from the same specification? I assume you were alphabetizing, but I don't think there's any order in this file.

Reply:
> Can we keep smaia and ssaia together since they come from the same specification? I assume you were alphabetizing, but I don't think there's any order in this file.

I've reordered it here and other places for consistency.